presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/PackageAPI.java

```diff
@@ -61,17 +64,20 @@
     public synchronized Response install(@NotNull @ApiParam("Url to fetch package") String locationToFetchPackage,
             @QueryParam("checkDependencies") @DefaultValue("true") @ApiParam("If false, disables dependency checking") boolean checkDependencies)
     {
+        LOGGER.debug("PUT /package  url: {} checkDependencies: {}", locationToFetchPackage, checkDependencies);
```

Comment:
There should be some indicator of the end of each parameter. I recommend putting a semicolon or comma after each `{}`.